packages/rollup-plugin/src/index.js

```diff
@@ -59,11 +60,21 @@ export default function stylexPlugin({
           useCSSLayers,
         );
 
+        // Process the CSS using lightningcss
+        const { code } = transform({
+          filename: 'stylex.css',
+          code: Buffer.from(collectedCSS),
+          minify: true,
```

Comment:
remove `minify: true`, but enable "syntax lowering". You can use a default value for `browsers` but the user should be able to override it.

Reply:
I think "syntax lowering" will be enabled when we add the supported browsers in the vendor list
https://lightningcss.dev/transpilation.html#syntax-lowering

Reply:
Yes. That's what I'm saying. You should choose a good default list for browsers to support. Perhaps `>0.5%` like the example in the docs? But then users should be able to customise it with their own list of supported browsers.

Reply:
Synced. Setting this to >1.0